### PR TITLE
SigLens VS ClickHouse 

### PR DESCRIPTION
**Pull request by sonamgupta21**

### CODE REVIEW

File: assets/sigscalr-vs-clickhouse/blog-cover.jpg
Comment (nkunal) [inline assets/sigscalr-vs-clickhouse/blog-cover.jpg:1]: remove this file, since it is not used

File: assets/sigscalr-vs-clickhouse/image4.png
Comment (nkunal) [inline assets/sigscalr-vs-clickhouse/image4.png:1]: remove this file, since it is not used

File: blog/sigscalr-vs-clickhouse.html [hunk 1]
```diff
@@ -1,664 +1,28 @@
...
-    <!-- Font-awesome Family -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
-        integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA=="
-        crossorigin="anonymous" referrerpolicy="no-referrer" />
-
-    <!-- Include Prism.js CSS -->
-    <link rel="stylesheet" href="./dark-theme-prismjs.css">
-
-    <link rel="canonical" href="https://www.sigscalr.io/blog/sigscalr-vs-clickhouse.html">
+    <link rel="canonical" href="https://www.siglens.com/blog/sigscalr-vs-clickhouse.html">
```
Comment (nkunal) [inline blog/sigscalr-vs-clickhouse.html:5]: shouldn't this point to `siglens-54x-faster....... html` ?

File: blog/sigscalr-vs-clickhouse.html [hunk 2]
```diff
@@ -1,664 +1,29 @@
...
     <!--OG Tags-->
-    <meta property="og:url" content="https://www.sigscalr.io/blog/sigscalr-vs-clickhouse.html" />
-    <meta property="og:title" content="One Billion Taxi Rides: Sigscalr vs ClickHouse" />
-    <meta property="og:description" content="SigScalr — a powerful solution designed to handle massive data volumes." />
+    <meta property="og:url" content="https://www.siglens.com/blog/sigscalr-vs-clickhouse.html" />
```
Comment (nkunal) [inline blog/sigscalr-vs-clickhouse.html:7]: replace this url, else infinite loop

**Review by nkunal — APPROVED** [review]

File: blog/sigscalr-vs-clickhouse.html (hunk 2 above)
Comment (sonamgupta21) [reply blog/sigscalr-vs-clickhouse.html]: updated the url to `siglens-54x-faster-than-clickhouse.html`

**Review by nbhavana — APPROVED** [review]
lgtm

**Pull request merged**